src/ChargeTrapping/ChargeTrapping.jl

```diff
@@ -375,6 +387,7 @@ function CombinedChargeTrappingModel{T}(config_dict::AbstractDict = Dict(); temp
     inactive_charge_trapping_model = if haskey(tmp, "model_inactive") && tmp["model_inactive"] == "ConstantLifetime"     
         ConstantLifetimeChargeTrappingModel{T}(tmp)
     else
+        if !isnothing(config_dict["model_inactive"]) @warn "Unknown inactive charge trapping model, running `NoChargeTrappingModel` inside inactive" end
```

Comment:
I would argue that we might wanna use the same charge trapping model as the bulk here as fallback default (?) instead of `NoChargeTrappingModel`.